Cargo.toml

```diff
@@ -58,7 +58,7 @@ features = ["display_link"]
 parking_lot = "0.11"
 
 [target.'cfg(target_os = "windows")'.dependencies.winapi]
-version = "0.3.6"
+version = "0.3.9"
```

Comment:
I assume you're bumping the version to get access to items which were previously unavailable?

Reply:
yes, `drop_handler.rs` defined a couple of missing structs that are now available.

Reply:
I meant 'dark_mode.rs`.